src/align.cpp

```diff
@@ -21,13 +21,13 @@
 static chunk_t *align_var_def_brace(chunk_t *pc, size_t span, size_t *nl_count);
 static chunk_t *align_trailing_comments(chunk_t *start);
 static void align_init_brace(chunk_t *start);
-static void align_func_params();
+static void align_func_params(void);
```

Comment:
I'm curious as to why you are adding ```void``` to these prototypes.
It seems the C++ way to is leave that out.

http://stackoverflow.com/questions/7412274/why-add-void-to-method-parameter-list